### PR TITLE
Imagebuild

### PR DESCRIPTION
**Pull request by Ganbingkun**
### What type of PR is this?

/kind design
/kind api-change
/kind failing-test

### What this PR does / why we need it:

- add S2I feature in DevOps module

new Api list：

- GET  /namespaces/{namespace}/imagebuildStrategies       // Get all ImageBuildStrategies
- POST  /namespaces/{namespace}/imagebuilds/{imagebuild}   // Create an ImageBuild
- GET  /namespaces/{namespace}/imagebuilds      // Get all imageBuilds
- GET /namespaces/{namespace}/imagebuilds/{imagebuild}   // Get an ImageBuild
- DELETE  /namespaces/{namespace}/imagebuilds/{imagebuild}   // Delete an ImageBuild
- PUT  /namespaces/{namespace}/imagebuilds/{imagebuild}   // Update an ImageBuild
- POST /namespaces/{namespace}/imagebuilds/{imagebuild}/imagebuildRuns  // Create an ImageBuildRun
- GET  /namespace/{namespace}/imagebuildRuns                  // Get all imageBuildRuns
- GET  /namespace/{namespace}/imagebuildRuns/{ImageBuildRun}  // Get an imageBuildRun
- DELETE  /namespaces/{namespace}/imagebuildRuns/{ImageBuildRun}  // Delete an ImageBuildRun

```release-note
None
```

### CODE REVIEW

File: pkg/kapis/s2i/v2apha1/handler.go
Comment (yudong2015) [inline pkg/kapis/s2i/v2apha1/handler.go:1]: The word v2apha1 in package name should be v2alpha1 ;

Comment (Ganbingkun) [reply pkg/kapis/s2i/v2apha1/handler.go:1]: ok

**Pull request merged**